### PR TITLE
Fixed markdown lint error

### LINKED ISSUE #70: Markdown lint errors
https://github.com/brisbanesocialchess/awesome-social-chess/actions/runs/11545144649/job/32131720823?pr=68

### PR DESCRIPTION
**Pull request by Virtual4087**
Fixes #70 

I believe I've formatted cortibution.md properly and the workflow will run smoothly now. If the workflow detects any more errors, I'll fix it right away.

### CODE REVIEW

**Comment by Virtual4087** [comment]
Hey @jbampton I think your approval is needed for pre-commit.yml to run

**Comment by Virtual4087** [comment]
The extra commits are because I tried to run the workflow in my own fork but was unable to do so.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -45,36 +44,41 @@ Clone the forked repository to your local machine:
...
-    cd awesome-social-chess 
-    git checkout -b add-new-feature
+  ```bash
+  cd awesome-social-chess 
+  git checkout -b add-new-feature
+  ```
 
-    ```
-### 4. ✏️ Make Changes: Make necessary improvements, such as fixing bugs, enhancing documentation, or adding new features.
+### 4. ✏️ Make Changes
```
Comment (jbampton) [inline CONTRIBUTING.md:54]:
```suggestion
### 4. ✏️ Make Changes - Make necessary improvements, such as fixing bugs, enhancing documentation, or adding new features.
```

Comment (Virtual4087) [reply CONTRIBUTING.md:54]: @jbampton  Got it

**Comment by Virtual4087** [comment]
@jbampton I believe it's good now

**Comment by Virtual4087** [comment]
@jbampton Could you explain the error that pre-commit is throwing rn?

**Review by jbampton — APPROVED** [review]

**Comment by jbampton** [comment]
Just got back from the supermarket.

Thanks once again !!

**Comment by jbampton** [comment]
> @jbampton Could you explain the error that pre-commit is throwing rn?

Was just some trailing whitespace in the markdown files

**Pull request merged**